beacon-chain/sync/initial-sync/service.go

```diff
@@ -513,7 +514,13 @@ func (s *InitialSync) validateAndSaveNextBlock(ctx context.Context, block *pb.Be
 	}
 
 	// Send block to main chain service to be processed.
-	s.chainService.IncomingBlockFeed().Send(block)
+	beaconState, err := s.chainService.ReceiveBlock(block)
```

Comment:
Use the ReceiveBlock func directly instead of a feed

Reply:
Now that you call ReceiveBlock from the RPC, can you pass in the context so that we can trace the full request span end-to-end?